### PR TITLE
Various UI fixes

### PR DESCRIPTION
**Pull request by andre8244**
- Replace login image
- Remove global search bar (not yet implemented)
- Expand/collapse side menu items
- Add empty state to pages not yet implemented
- Move language selection inside Account page

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -80,25 +80,35 @@ const standaloneRoutes = [
...
+  {
+    path: 'report',
+    name: 'Report',
+    component: () => import('../views/standalone/ReportView.vue')
+  },
   {
     path: 'user',
     name: 'UserManagement',
     component: () => import('../views/standalone/UserManagement.vue')
-  },
```
Comment (Tbaile) [inline src/router/index.ts:97]: Are you sure about this removal?

File: src/components/standalone/StandaloneAppShell.vue [hunk 2]
```diff
@@ -270,7 +270,7 @@ watch(
 
         <div class="flex flex-1 gap-x-4 self-stretch lg:gap-x-6">
           <form class="relative flex flex-1" action="#" method="GET">
-            <label for="search-field" class="sr-only">{{ t('common.search') }}</label>
+            <!-- <label for="search-field" class="sr-only">{{ t('common.search') }}</label> //// global search
             <font-awesome-icon
               :icon="['fas', 'magnifying-glass']"
               :class="[
```
Comment (Tbaile) [inline src/components/standalone/StandaloneAppShell.vue:276]: Just remove it if it's commented out

File: src/components/standalone/StandaloneAppShell.vue [hunk 3]
```diff
@@ -285,7 +285,7 @@ watch(
               :placeholder="t('common.search')"
               type="search"
               name="search"
-            />
+            /> -->
```
Comment (Tbaile) [inline src/components/standalone/StandaloneAppShell.vue:288]: Same as above

File: src/components/standalone/StandaloneAppShell.vue (hunk 2 above)
Comment (andre8244) [reply src/components/standalone/StandaloneAppShell.vue:276]: It will be uncommented soon

File: src/router/index.ts (hunk 1 above)
Comment (andre8244) [reply src/router/index.ts:97]: It's not removed, it's moved at line 83

**Review by Tbaile — APPROVED** [review]

**Pull request merged**